### PR TITLE
[dataset][cuj2] add another single node ingestion example

### PR DESCRIPTION
**Pull request by scv119**
This is the example one of our open source user is using; reproduce it using ray train.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [x] Release tests https://console.anyscale.com/o/anyscale-internal/projects/prj_SVFGM5yBqK6DHCfLtRMryXHM/clusters/ses_zKhMWrD4yXF7NZ287CLqZ5Uq
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by amogkam** [comment]
@scv119 is the success criteria for these tests just that they finish running? Or do we measure performance regressions as well?

File: release/nightly_tests/dataset/dataset_test.yaml [hunk 1]
```diff
@@ -76,3 +76,18 @@
...
+
+  cluster:
+    app_config: ray_sgd_training_app.yaml
+    compute_template: dataset_ingest_400G_compute.yaml
+
+  run:
+    timeout: 7200
+    script: python ray_sgd_runner.py --address auto --use-gpu --num-epochs 1
+
+  stable: false
```
Comment (rkooo567) [inline release/nightly_tests/dataset/dataset_test.yaml:93]: Newline!

File: release/nightly_tests/dataset/dataset_ingest_400G_compute.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+cloud_id: cld_17WvYIBBkdgLwEUNcLeRAE
```
Comment (rkooo567) [inline release/nightly_tests/dataset/dataset_ingest_400G_compute.yaml:1]: Why hardcoding the cloud id?

File: release/nightly_tests/dataset/dataset_ingest_400G_compute.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+region: us-west-2
+allowed_azs:
```
Comment (rkooo567) [inline release/nightly_tests/dataset/dataset_ingest_400G_compute.yaml:3]: Remove it?

**Review by rkooo567 — COMMENTED** [review]
Only commented on testing config parts

File: release/nightly_tests/dataset/dataset_ingest_400G_compute.yaml (hunk 2 above)
Comment (scv119) [reply release/nightly_tests/dataset/dataset_ingest_400G_compute.yaml:1]: ah, to resolve read from aws permission issues.

**Comment by scv119** [comment]
@amogkam yes, we need to monitor the avg load/train time as well!

**Review by rkooo567 — APPROVED** [review]
Didn't review the test script

**Pull request merged**